blivet/osinstall.py

```diff
@@ -1340,10 +1359,93 @@ def _update_custom_storage_ksdata(self):
         self.fsset = FSSet(self.devicetree)
 
     def reset(self, cleanup_only=False):
+        """ Reset storage configuration to reflect actual system state.
+
+            This will cancel any queued actions and rescan from scratch but not
+            clobber user-obtained information like passphrases, iscsi config, &c
+
+            :keyword cleanup_only: prepare the tree only to deactivate devices
+            :type cleanup_only: bool
+
+            See :meth:`devicetree.Devicetree.populate` for more information
+            about the cleanup_only keyword argument.
+        """
+        log.info("resetting Blivet (version %s) instance %s", __version__, self)
```

Comment:
You can let `Blivet` handle logging this, which allows you to not import __version__ above.